components/file_upload/file_upload.test.jsx

```diff
@@ -98,6 +98,20 @@ describe('components/FileUpload', () => {
         expect(baseProps.onClick).toHaveBeenCalledTimes(1);
     });
 
+    test('should call onClick on fileInput when button is touched', () => {
+        const wrapper = shallowWithIntl(
+            <FileUpload {...baseProps}/>
+        );
+
+        wrapper.instance().fileInput = {
+            current: {
+                click: () => baseProps.onClick(),
+            },
+        };
+        wrapper.find('button').simulate('touchend');
+        expect(baseProps.onClick).toHaveBeenCalledTimes(1);
```

Comment:
``click: () => baseProps.onClick(),`` seems to be not exacly represented.  I feel like the test should be like:

```javascript
        const instance = wrapper.instance();
        instance.handleLocalFileUploaded = jest.fn();
        instance.fileInput = {
            current: {
                click: () => instance.handleLocalFileUploaded(),
            },
        };
        wrapper.find('button').simulate('touchend');
        expect(instance.handleLocalFileUploaded).toHaveBeenCalledTimes(1);
```

Reply:
Thanks Saturn, ready for re-review